### PR TITLE
Alerting: Fix saving telegram contact point to Cloud AM config

### PR DESCRIPTION
**Pull request by tomratcliffe**
**What is this feature?**

Fixes saving contact points when the user hadn't interacted with any present select dropdowns in the forms.
In this case, we hadn't mapped the `defaultValue` object to get its `value` property, and were sending an object to the API instead, e.g.

```ts
{
  //...
  parse_mode: {
    label: 'MarkdownV2',
    value: 'MarkdownV2',
  }
}
```

instead of 

```ts
{
  //...
  parse_mode: 'MarkdownV2',
}
```

We can map the default value to be the value property. The existing behaviour already handles mapping the selected option on change, hence why this would work if you changed the dropdown before saving

**Why do we need this feature?**

🐛 

**Who is this feature for?**

Alerting users creating/editing contact points - primarily Telegram/WeChat

### CODE REVIEW

File: public/app/features/alerting/unified/Receivers.tsx [hunk 1]
```diff
@@ -12,7 +11,7 @@ const EditContactPoint = SafeDynamicImport(() => import('./components/contact-po
 const NewContactPoint = SafeDynamicImport(() => import('./components/contact-points/NewContactPoint'));
 const GlobalConfig = SafeDynamicImport(() => import('./components/contact-points/components/GlobalConfig'));
 
-const ContactPoints = (_props: GrafanaRouteComponentProps): JSX.Element => (
```
Comment (soniaAguilarPeiron) [inline public/app/features/alerting/unified/Receivers.tsx:15]: I know this is not used now, but are we sure we want to delete these _props in this pr? maybe it was here on purpose for some reason?

**Review by soniaAguilarPeiron — APPROVED** [review]
LGTM!

File: public/app/features/alerting/unified/Receivers.tsx (hunk 1 above)
Comment (tomratcliffe) [reply public/app/features/alerting/unified/Receivers.tsx:15]: Fair question - I removed it here as in the tests I then have to pass in all sorts of route props (route, queryParams, history, location, match), and they weren't used before this anyway. I had to make a similar change in this PR https://github.com/grafana/grafana/pull/88487/files#diff-5e4993ba6405cb0c042ccd8d131bd81d954ff9a86d4b854b1cc2e7f095f798ab

File: public/app/features/alerting/unified/Receivers.test.tsx [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+  // Fill out basic fields necessary for contact point to be saved
+  const botToken = await screen.findByLabelText(/bot token/i);
+  const chatId = await screen.findByLabelText(/chat id/i);
+
+  await user.type(botToken, 'sometoken');
+  await user.type(chatId, '-123');
+
+  await user.click(await screen.findByRole('button', { name: /save contact point/i }));
+
+  // TODO: Have a better way to assert that the contact point was saved. This is instead asserting on some
```
Comment (soniaAguilarPeiron) [inline public/app/features/alerting/unified/Receivers.test.tsx:71]: We could check that the error message is not present?

**Pull request merged**